src/hooks/validators/getNumberValidator.ts

```diff
@@ -0,0 +1,140 @@
+import { ValidationOptions } from 'react-hook-form'
+
+import { JSONSchemaType } from '../../JSONSchema'
+import { ErrorTypes } from './types'
+
+// Used for exclusiveMinimum and exlusiveMaximum values
+const epsilon = 0.0001
+
+export const toFixed = (value: number, precision: number): string => {
```

Comment:
I saw the implementation of this function in multiple places, maybe reusing would be better

Reply:
Why not `number.toFixed()`? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed

Reply:
Because I didn't want internal precision to interfere with the rounding:
```js
(49.175).toFixed(2) => "49.17"
(49.775).toFixed(2) => "49.77"
(49.185).toFixed(2) => "49.19"
(49.785).toFixed(2) => "49.78"

(49.1175).toFixed(3) => "49.117"
(49.1775).toFixed(3) => "49.178"
(49.1185).toFixed(3) => "49.118"
(49.1785).toFixed(3) => "49.178"
```